contractsdk/cpp/XRC/xrc01/src/main/xrc01.h

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include "xchain/xchain.h"
+#include "table/types.h"
+#include "table/table.tpl.h"
+#include "pb/xrc01.pb.h"
+
+// XRC01 is the implement of XRC-01 multi token standard
+class XRC01 {
```

Comment:
if this is our proposal for token issuing, we should provide an interface first, then is our implementation.

Reply:
Good idea, but this has been reviewed internally for several times.  In the future, we can define a process to propose XRC standard. I will write a wiki for this.